kolibri/plugins/learn/assets/src/state/actions.js

```diff
@@ -259,17 +259,18 @@ function showLearnChannel(store, channelId, page = 1) {
         router.replace({ name: constants.PageNames.CONTENT_UNAVAILABLE });
         return;
       }
+      const isFacilityUser = coreGetters.isFacilityUser(store.state);
       const nextStepsPayload = { next_steps: session.user_id };
-      const popularPayload = { popular: session.user_id };
+      const popularPayload = { popular: 'true' };
```

Comment:
Possibly needs a little cleanup in the `ContentNodeFilter` to remove the reference to `user_id` in comments, but not a blocker.